### ISSUE TITLE
Add-on store: translations aren't always working for Spanish in the available tab

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#15722]
### Steps to reproduce:
1. Open the store.
2. Go to Available add-ons.
3. Search for reportSymbols stable (currently 19.0.0version).
4. Press tab to read the add-on description.
### Actual behavior:

The description  is shown in English, as reflected below:
> Complementos disponibles:  lista
10dita application module; Versión disponible: 2023.1; Canal: Estable; Editor: Alberto Zanella; Estado: Disponible  1 de 140
Read Feeds; Versión disponible: 25.0.0; Canal: Estable; Editor: nvdaes; Estado: Disponible  86 de 140
Report Passwords; Versión disponible: 11.0.0; Canal: Estable; Editor: nvdaes; Estado: Disponible  91 de 140
NVDA+l
Report Passwords; Versión disponible: 11.0.0; Canal: Estable; Editor: nvdaes; Estado: Disponible
flecha abajo
Report Symbols; Versión disponible: 19.0.0; Canal: Estable; Editor: nvdaes; Estado: Deshabilitado  92 de 140
tab
Descripción:  edición  sólo lectura  multilínea  Allows to listen the typed symbols (non alphanumeric or blank characters), even when the speaking of characters 
ctrl+inicio
Allows to listen the typed symbols (non alphanumeric or blank characters), even when the speaking of characters 
flecha abajo
is turned off.

### Expected behavior:
The description should be presented in Spanish.

### NVDA logs, crash dumps and other attachments:
N/a
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA 2023.3rc2
#### Windows version:
Windows 11 22H2 (10.0.22621) workstation AMD64

#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. In NVDA alpha-29812,da446b38 (2024.1.0.29812), the behavior is the same.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

### Additional note

In the views branch of the store, the description appears in English. Please see views/es/latest/reportSymbols/stable.json

**Comment by nvdaes** [comment]
cc @CyrilleB79 , in case you are interested in testing.

**Comment by ruifontes** [comment]
Noelia, here the text appears in portuguese...
    
    
    Rui Fontes
    
    
    Às 18:52 de 31/10/2023, Noelia Ruiz
      Martínez escreveu:
    
    
      
      Steps to reproduce:
      
        Open the store.
        Go to Available add-ons.
        Search for reportSymbols stable (currently 19.0.0version).
        Press tab to read the add-on description.
      
      Actual behavior:
      The description is shown in English, as reflected
        below:
      
        Complementos disponibles: lista
          10dita application module; Versión disponible: 2023.1; Canal:
          Estable; Editor: Alberto Zanella; Estado: Disponible 1 de 140
          Read Feeds; Versión disponible: 25.0.0; Canal: Estable;
          Editor: nvdaes; Estado: Disponible 86 de 140
          Report Passwords; Versión disponible: 11.0.0; Canal: Estable;
          Editor: nvdaes; Estado: Disponible 91 de 140
          NVDA+l
          Report Passwords; Versión disponible: 11.0.0; Canal: Estable;
          Editor: nvdaes; Estado: Disponible
          flecha abajo
          Report Symbols; Versión disponible: 19.0.0; Canal: Estable;
          Editor: nvdaes; Estado: Deshabilitado 92 de 140
          tab
          Descripción: edición sólo lectura multilínea Allows to listen
          the typed symbols (non alphanumeric or blank characters), even
          when the speaking of characters
          ctrl+inicio
          Allows to listen the typed symbols (non alphanumeric or blank
          characters), even when the speaking of characters
          flecha abajo
          is turned off.
      
      Expected behavior:
      The description should be presented in Spanish.
      NVDA logs, crash dumps and other attachments:
      N/a
      System configuration
      NVDA installed/portable/running from source:
      Installed
      NVDA version:
      NVDA 2023.3rc2
      Windows version:
      Windows 11 22H2 (10.0.22621) workstation AMD64
      Name and version of other software in use when
        reproducing the issue:
      N/A
      Other information about your system:
      N/A
      Other questions
      Does the issue still occur after restarting your
        computer?
      Yes
      Have you tried any other versions of NVDA? If so,
        please report their behaviors.
      Yes. In NVDA alpha-29812,da446b38 (2024.1.0.29812),
        the behavior is the same.
      If NVDA add-ons are disabled, is your problem still
        occurring?
      Yes
      Does the issue still occur after you run the COM
        Registration Fixing Tool in NVDA's tools menu?
      Yes.
      Additional note
      In the views branch of the store, the description
        appears in English. Please see
        views/es/latest/reportSymbols/stable.json
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you are subscribed to this
        thread.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/nvaccess/nvda/issues/15722",
"url": "https://github.com/nvaccess/nvda/issues/15722",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by nvdaes** [comment]
@ruifontes thanks, I realised that pt is working and even changed NVDA to this language to confirm it. I don"t know what is happening with spanish; seems some kind of issue in the transform GitUb workflow in the addondatastore repo, but in case I open this issue here.

**Comment by seanbudd** [comment]
@nvdaes - the description at [es/latest/reportSymbols/stable.json](https://github.com/nvaccess/addon-datastore/blob/views/views/es/latest/reportSymbols/stable.json) is in spanish for me.

I also cannot reproduce this issue in NVDA.

The speech you copied into the issue description also appears in spanish. However, an English description is rendered after `ctrl+inicio` what is this command?

I am closing this as works for me

**Comment by nvdaes** [comment]
@seanb, control+inicio is control+home, to go to the start of the description, here presented just in English. Other info corresponds to the list item. Bu luckily now this is fixed and I cannot reproduce the bug anymore. Also, previously I've tested using the webpage created by me available at

https://nvdaes.github.io/nvdastore

There I set the language to es and the description was presented in English, and in the views branch, previously, the description was also in English. Maybe that a transformation workflow has been performed due to the submission of other add-on, but now this works. I don't know what was the problem.
